### PR TITLE
Extended the LowDiscrepancyExperiment class to dependent distributions

### PR DESCRIPTION
**Pull request by regislebrun**
This extension is based on the idea presented in:
Mathieu Cambou, Marius Hofert, Christiane Lemieux, "Quasi-Random numbers for copula models", Statistics and Computing, September 2017, Volume 27, Issue 5, pp 1307–1329

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -119,6 +119,7 @@
 === Miscellaneous ===
  * Multivariate TruncatedDistribution
  * Asymptotic Sobol' variance estimators
+ * LowDiscrepancyExperiment to work with dependent distributions
```
Comment (sofianehaddad) [inline ChangeLog]: please mention also ticket 949

File: python/src/LowDiscrepancyExperiment_doc.i.in [hunk 2]
```diff
@@ -31,8 +30,9 @@ restart : bool
 Notes
 -----
 The :meth:`generate` method generates points :math:`(\Xi_i)_{i \in I}`
-independently from the distribution :math:`\mu`. When the :meth:`generate`
-method is recalled, the generated sample changes.
+according to the distribution :math:`\mu`. When the :meth:`generate`
+method is recalled, the generated sample changes. In case of dependent
```
Comment (dbarbier) [inline python/src/LowDiscrepancyExperiment_doc.i.in]: s/recalled/called again/

File: python/src/LowDiscrepancyExperiment_doc.i.in [hunk 3]
```diff
@@ -96,6 +96,18 @@ of distribution:
 2 : [ 0.8125 0.6875 ]
 3 : [ 0.5625 0.4375 ]
 4 : [ 0.0625 0.9375 ]
+
+Generate a sample according to a distribution with depedent marginals:
```
Comment (dbarbier) [inline python/src/LowDiscrepancyExperiment_doc.i.in]: Typo: s/depedent/dependent/

**Comment by regislebrun** [comment]
Ok done.

**Pull request merged**